app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,2 @@
+class Api::V1::OrganizationsController < ApplicationController
```

Comment:
Don't forget to run 'bundle exec rubocop` to check some "offences" in the file.

Reply:
You would need to add the action `show` with the corresponding logic inside.

Reply:
@givaleria

Reply:
@Ezequiel-Padilla I've already changed it in another commit

Reply:
Please @givaleria, check the syntax executing `bundle exec rubocop` in terminal.